### DISCUSSION TITLE
It would be helpful to be able to set a Main Scene for Export in addition to just a Main scene

### DISCUSSION THREAD

**Discussion by blackears** [godotengine/godot-proposals#4703] [Editor]
At the moment, when Godot exports a project, it uses the main scene at the first file to run when the game is launched.  However, when I'm developing I'll usually set the main scene to something other than the scene that should be the starting point for the exported project - there's not point to always launching the title screen when you're debugging a level midway into the game.  However this means that when you build for export you need to change your main scene back to the title screen, do your export and then change it back again to keep developing.  

It would be very helpful if it were possible to set an override for the main scene for when you're exporting so you don't have to keep manually switching your main scene whenever you do an export.

**Comment by Calinou** [comment]
This can already be done with [feature tags](https://docs.godotengine.org/en/stable/getting_started/workflow/export/feature_tags.html) by overriding the `application/run/main_scene` project setting with the `standalone` feature tag.

You can also add a command line argument to your game to allow starting a level directly (parse `OS.get_cmdline_args()`), and use the  to `editor/main_run_args` project setting to specify this argument. It will only be used when running the project from the editor.
